CONTRIBUTING.md

```diff
@@ -136,43 +156,51 @@ $ git remote add upstream git://github.com/stdlib-js/stdlib.git
 
 For modifications intended to be included in stdlib, create a new local branch.
 
+<!-- run-disable -->
+
 ```bash
 $ git checkout -b <branch>
 ```
 
-where `<branch>` is the branch name. Both the `master` and `develop` branches for the main stdlib project are protected, and direct modifications to these branches will **not** be accepted. Instead, all contributions should be made on non-master and non-develop local branches, including documentation changes and other non-code modifications.
+where `<branch>` is the branch name. Both the `master` and `develop` branches for the main stdlib project are protected, and direct modifications to these branches will **not** be accepted. Instead, all contributions should be made on non-master and non-develop local branches, including documentation changes and other non-code modifications. See the project [branching guide][stdlib-branching] for additional guidance.
 
 #### Step 3: Write
 
-Start making your changes and/or implementing the new feature. Any text you write should follow the [text style guide][stdlib-style-guides], including comments and API documentation.
+Start making your changes and/or implementing the new feature. Any text you write should follow the [text style guide][stdlib-style-guides-text], including comments and API documentation.
 
 #### Step 4: Commit
 
 Ensure that you have configured [Git][git] to know your name and email address.
 
+<!-- run-disable -->
+
 ```bash
 $ git config --global user.name "Jane Doe"
 $ git config --global user.email "jane.doe@example.com"
 ```
 
 Add changed files and commit.
 
+<!-- run-disable -->
+
 ```bash
 $ git add files/which/changed
 $ git commit
 ```
 
-When writing commit messages, follow the Git [style guide][stdlib-style-guides].
+When writing commit messages, follow the Git [style guide][stdlib-style-guides-git]. Adherence to project commit conventions is necessary for project automation which automatically generates release notes and changelogs from commit messages.
```

Comment:
Should add a comma before "which".

Reply:
I think we can omit the comma here, as the clause following `which` is necessary for which project automation conventions are necessary. A simple test is whether the sentence would make sense without the `which` clause. In this case, the sentence would no longer be accurate as, in general, conventional commits is not necessary for **all** project automation, but only for automation which generates release notes and changelogs.